packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewerContainer.spec.js

```diff
@@ -1,199 +1,186 @@
 import { expect } from 'chai'
-import { mount } from 'enzyme'
-import nock from 'nock'
+import { shallow } from 'enzyme'
 import React from 'react'
 import { Factory } from 'rosie'
 import sinon from 'sinon'
+import asyncStates from '@zooniverse/async-states'
 
 import SingleTextViewerContainer from './SingleTextViewerContainer'
 import SingleTextViewer from './SingleTextViewer'
 
-
 const subject = Factory.build('subject', {
+  content: 'subject text',
+  contentLoadingState: asyncStates.success,
   locations: [
-    { 'text/plain': 'http://localhost:8080/subject.txt' }
+    { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
   ]
 })
 
 const nextSubject = Factory.build('subject', {
+  content: 'next subject text',
+  contentLoadingState: asyncStates.success,
+  locations: [
+    { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
+  ]
+})
+
+const errorSubject = Factory.build('subject', {
+  contentLoadingState: asyncStates.error,
+  error: new Error('Error loading subject'),
   locations: [
-    { 'text/plain': 'http://localhost:8080/nextSubject.txt' }
+    { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
   ]
 })
 
-const imageSubject = Factory.build('subject')
+const initializedSubject = Factory.build('subject', {
+  contentLoadingState: asyncStates.initialized,
+  locations: [
+    { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
+  ]
+})
 
-const failSubject = Factory.build('subject', {
+const loadingSubject = Factory.build('subject', {
+  contentLoadingState: asyncStates.loading,
   locations: [
-    { 'text/plain': 'http://localhost:8080/failure.txt' }
+    { 'text/plain': 'http://localhost:8080/subjectContent.txt' }
   ]
 })
 
 describe('Component > SingleTextViewerContainer', function () {
   it('should render without crashing', function () {
-    const wrapper = mount(<SingleTextViewerContainer />)
+    const wrapper = shallow(<SingleTextViewerContainer />)
```

Comment:
Full DOM mounting here will make it possible to move these tests from Enzyme to React Testing Library.